### PR TITLE
Fix x64 release build

### PR DESCRIPTION
**Pull request by SiLeader**
+ fix project files
  + link with missing libraries
  + add preprocessor definitions
+ include the lacked include files
+ fix include directive near the "#ifdef DBG"

### CODE REVIEW

**Comment by cezuni** [comment]
@SiLeader : Your patch is very helpful but release build is not a high priority job. As you know, usbip-win is premature. 😄 Another reason for not merging right now is that toolset update (#41) is under way.

**Comment by SiLeader** [comment]
@cezuni 
thank you for your comment.
please merge this PR when you need it.

File: driver/stub/usbip_stub.vcxproj [hunk 1]
```diff
@@ -178,6 +178,7 @@
       <WppScanConfigurationData Condition="'%(ClCompile.ScanConfigurationData)' == ''">trace.h</WppScanConfigurationData>
       <WppKernelMode>true</WppKernelMode>
       <AdditionalIncludeDirectories>..\..\include;..\lib;$(IntDir);%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <TreatWarningAsError>false</TreatWarningAsError>
```
Comment (cezuni) [inline driver/stub/usbip_stub.vcxproj]: Please remove all warnings without this option.

**Comment by cezuni** [comment]
@SiLeader : I have cherry-picked your db14a4c7ee231c5e38ccf68f87f26ad75584daf8. Also additional commit 4b2d7cefe506cfdef9deafca94a64d9ed722a560 is added to remove compilation warnings for release configuration.

**Pull request closed without merging**